wire-library/wire-schema-tests/src/test/java/com/squareup/wire/recipes/ErrorReportingCustomHandler.kt

```diff
@@ -0,0 +1,30 @@
+package com.squareup.wire.recipes
+
+import com.squareup.wire.schema.AbstractSchemaHandler
+import com.squareup.wire.schema.Extend
+import com.squareup.wire.schema.Field
+import com.squareup.wire.schema.MessageType
+import com.squareup.wire.schema.SchemaHandler
+import com.squareup.wire.schema.Service
+import com.squareup.wire.schema.Type
+import okio.Path
+
+class ErrorReportingCustomHandler : AbstractSchemaHandler() {
```

Comment:
Add a comment:

/** Sample schema validator that enforces a field naming pattern. */